cloudos/jobs/job.py

```diff
@@ -275,6 +281,28 @@ def convert_nextflow_to_json(self,
             if len(workflow_params) == 0:
                 raise ValueError(f'The {job_config} file did not contain any ' +
                                  'valid parameter')
+        if len(parameter) > 0:
+            for p in parameter:
+                p_split = p.split('=')
+                if len(p_split) != 2:
+                    raise ValueError('Please, specify -p / --parameter using a single \'=\' ' +
+                                     'as spacer. E.g: input=value')
+                p_name = p_split[0]
+                p_value = p_split[1]
+                if workflow_type == 'wdl':
+                    param = {"prefix": "",
+                             "name": p_name,
+                             "parameterKind": "textValue",
+                             "textValue": p_value}
+                    workflow_params.append(param)
+                else:
```

Comment:
This is implicit (aka a bit anti-pythonic 😄 ), it says "**when workflow is not WDL"**, but unless we read the if clause, we don't know what that means.

Additionally, in the future we will support a third type of workflow, `docker` for Docker Batch jobs in CloudOS.
eg https://cloudos.lifebit.ai/public/jobs/61793a0888e0c901db2e3603.

In that case you would need to be explicit and reimplement as 
```Suggestion
                if workflow_type == 'nextflow':
```

This would be extended for all the job types we will support in the future.

Yoda says:
https://speakerdeck.com/jennybc/code-smells-and-feels?slide=45
<img width="903" alt="image" src="https://user-images.githubusercontent.com/38183826/184122508-ee37e0d0-0e42-42ce-81fd-333c96a32229.png">